### ISSUE TITLE
PHP 7.0 not found during Vagrant up

### ISSUE THREAD

**Issue by zwrawr** [ludumdare/dairybox#58]
Looks like there is something wrong with either the php7.0 package or its PPA.

This resulting in the VM serving the PHP files as plain text rather than running them, breaking the toolchain completely.

## errors during vagrant up
![Screenshot from 2019-05-18 23-31-45](https://user-images.githubusercontent.com/18352787/57975668-2652f580-79c5-11e9-9f7e-59d2852e396d.png)

```
  default: Fetched 23.2 kB in 2s (10.0 kB/s)
    default: Reading package lists...
    default: W: An error occurred during the signature verification. The repository is not updated and the previous index files will be used. GPG error: http://packages.blackfire.io any InRelease: The following signatures couldn't be verified because the public key is not available: NO_PUBKEY 696DBE66A72D76DA
    default: 
    default: W: Failed to fetch http://packages.blackfire.io/debian/dists/any/InRelease  
    default: 
    default: W: Some index files failed to download. They have been ignored, or old ones used instead.
    default: Reading package lists...
    default: Building dependency tree...
    default: 
    default: Reading state information...
    default: E: Unable to locate package php7.0
    default: E: Couldn't find any package by regex 'php7.0'
    default: E: Unable to locate package php7.0-mbstring
    default: E: Couldn't find any package by regex 'php7.0-mbstring'
    default: E: Unable to locate package php7.0-mysql
    default: E: Couldn't find any package by regex 'php7.0-mysql'
    default: E: Unable to locate package php7.0-xml
    default: E: Couldn't find any package by regex 'php7.0-xml'
    default: E: Unable to locate package php7.0-opcache
    default: E: Couldn't find any package by regex 'php7.0-opcache'
    default: E: Unable to locate package php7.0-gd
    default: E: Couldn't find any package by regex 'php7.0-gd'
    default: E: Unable to locate package php7.0-curl
    default: E
    default: : Couldn't find any package by regex 'php7.0-curl'
    default: E: Unable to locate package php7.0-zip
    default: E: Couldn't find any package by regex 'php7.0-zip'
    default: E: Unable to locate package php7.0-redis
    default: E: Couldn't find any package by regex 'php7.0-redis'
    default: E: Unable to locate package php-apcu
    default: Module php5 disabled.
    default: To activate the new configuration, you need to run:
    default:   service apache2 restart
    default: ERROR: Module php7.0 does not exist!
    default: cp: cannot create regular file ‘/etc/php/7.0/apache2/conf.d/’: No such file or directory
    default: cp: cannot create regular file ‘/etc/php/7.0/apache2/conf.d/user.ini’
    default: : No such file or directory

```

**Comment by zwrawr** [comment]
Possibly related to https://github.com/laravel/homestead/issues/1009 but I do not get the prompt that is referenced.
Adding the key from https://blackfire.io/docs/up-and-running/installation manually allowed `sudo apt update` to run.

**Comment by zwrawr** [comment]
So this one might not be so easy to fix.
ubuntu 14 is no longer a supported version. standard support expired in April 2019.
This means ppa:ondj/php no longer provides packages for ubuntu 14.

**Comment by zwrawr** [comment]
changing the php version in bootstrap.sh to 5 will at least allow php to be installed and serve the site. But may brake other things.

**Comment by mikekasprzak** [comment]
Resolved (now PHP 7.1)
